### PR TITLE
Feature/settings2.1

### PR DESCRIPTION
**Pull request by villenavg**

### CODE REVIEW

File: OWDTestToolkit/DOM/OMACP.py [hunk 1]
```diff
@@ -11,3 +11,4 @@
 CP_Cancel_Button = ("xpath", '//form[@id="cp-store-confirm"]//button[@data-l10n-id="cancel"]')
 CP_Cancel_Message = ("xpath", '//form[@id="cp-quit-app-confirm"]//strong[@data-l10n-id="cp-quit-app-confirm-dialog-message"]')
 CP_Quit_Button = ("xpath", '//form[@id="cp-quit-app-confirm"]//button[@data-l10n-id="quit"]')
+CP_Confirm_Cancel = ("xpath", '//form[@id="cp-quit-app-confirm"]//button[@data-l10n-id="cancel"]')
```
Comment (FranciscoCano) [inline OWDTestToolkit/DOM/OMACP.py]: what about a css locator?

    ("css selector", "#cp-quit-app-confirm button[data-l10n-id=cancel]")

simpler, nicer.

Comment (villenavg) [inline OWDTestToolkit/DOM/OMACP.py]: Sloth

File: OWDTestToolkit/apps/dialer.py [hunk 2]
```diff
@@ -412,7 +412,8 @@ def hangUp(self):
             self.UTILS.iframe.switchToFrame(*DOM.Dialer.frame_locator_calling)
             self.marionette.find_element(*DOM.Dialer.hangup_bar_locator).tap()
             self.apps.switch_to_displayed_app()
-        except:
+        except Exception, e:
```
Comment (FranciscoCano) [inline OWDTestToolkit/apps/dialer.py]: Syntax:

    except Exception as e:

is preferred in modern Python versus the comma-based old one.

File: OWDTestToolkit/apps/settings.py [hunk 3]
```diff
@@ -48,60 +69,143 @@ def call_settings(self, sim_card_number=1):
...
-    def three_times_bad_pin2(self, wrong_pin2):
-        """Change the PIN2.
-
-        Enter a wrong PIN2 three consecutive times. After that, the PUK2 is requested,
-        and also a new PIN2 value is entered twice.
-        """
-        for i in range(3):
-            self.fdn_type_pin2(wrong_pin2)
-
-    def restore_pin2(self, good_pin2, puk2):
```
Comment (FranciscoCano) [inline OWDTestToolkit/apps/settings.py]: Why removing these two auxiliary methods?

File: OWDTestToolkit/apps/settings.py [hunk 4]
```diff
@@ -48,60 +69,143 @@ def call_settings(self, sim_card_number=1):
...
+            DOM.Settings.networkOperator_OK_btn, "Network Operator. Click on OK button")
+        select_ok_btn.tap()
+        self.UTILS.iframe.switchToFrame(*DOM.Settings.frame_locator)
+
+    def confirm_data_conn(self):
+        self.UTILS.iframe.switchToFrame(*DOM.Settings.frame_locator)
+        try:
+            self.UTILS.reporting.logResult("info", "Waiting for data switch-on confirmation")
+            self.parent.wait_for_element_displayed(*DOM.Settings.celldata_DataConn_ON)
+            x = self.marionette.find_element(*DOM.Settings.celldata_DataConn_ON)
```
Comment (FranciscoCano) [inline OWDTestToolkit/apps/settings.py]: Uuuuuuuuhhhhhhhhh

File: OWDTestToolkit/apps/settings.py [hunk 5]
```diff
@@ -371,6 +457,33 @@ def fdn_delete_all_auth_numbers(self, pin2):
             self.fdn_delete_auth_number(number, pin2)
             time.sleep(2)
 
+    def three_times_bad_pin2(self, wrong_pin2):
```
Comment (FranciscoCano) [inline OWDTestToolkit/apps/settings.py]: Just moving these methods :+1:

File: OWDTestToolkit/apps/dialer.py (hunk 2 above)
Comment (villenavg) [inline OWDTestToolkit/apps/dialer.py]: This shouldn't be here. Don't really know why is there, though.

File: OWDTestToolkit/apps/settings.py (hunk 4 above)
Comment (villenavg) [inline OWDTestToolkit/apps/settings.py]: Cristiano Ronaldo?

**Comment by FranciscoCano** [comment]
LGTM

**Pull request merged**